internal/admin/handler.go

```
@@ -270,14 +270,10 @@ func (h *Handler) CreateUser(c *gin.Context) {
 
 func getUserName(c *gin.Context) (string, error) {
-	encodedUsername := c.Param("username")
-	username, err := common.DecodeFromBase64(encodedUsername)
-	if err != nil {
-		common.ErrorWithCode(c, common.CodeBadRequest, err.Error())
-		return "", err
-	}
+	username := c.Param("username")
 	if username == "" {
-		common.ErrorWithCode(c, common.CodeBadRequest, "Username is required")
+		err := errors.New("Username is required")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)